src/Microsoft.ML.DataView/ImageDataViewType.cs

```diff
@@ -3,11 +3,9 @@
 // See the LICENSE file in the project root for more information.
 
 using System.Drawing;
-using Microsoft.ML.Data;
-using Microsoft.ML.Internal.Utilities;
-using Microsoft.ML.Runtime;
+using Microsoft.ML.Internal.DataView;
 
-namespace Microsoft.ML.Transforms.Image
+namespace Microsoft.ML.Data
```

Comment:
These kinds of changes are API breaking. We can't be making these changes after 1.0.